src/App.js

```diff
@@ -1,9 +1,11 @@
 import './App.css';
+import './componentsCss/calculatorGrid.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] According to the project, the `App.js` component should be a class, please fix it (in the next steps you'll see why). 🐱